Doc/library/uuid.rst

```diff
@@ -288,11 +288,12 @@ The following options are accepted:
    Specify the function name to use to generate the uuid. By default :func:`uuid4`
    is used.
 
-.. cmdoption:: -ns <namespace>
+.. cmdoption:: -n <namespace>
                --namespace <namespace>
 
-   The namespace used as part of generating the uuid. Only required for
-   :func:`uuid3` / :func:`uuid5` functions.
+   The namespace is a ``UUID``, or ``@ns`` where ``ns`` is a well-known predefined UUID
+   addressed by namespace name. Such as ``@dns``, ``@url``, ``@oid``, and ``@x500``.
+   Only required for :func:`uuid3` / :func:`uuid5` functions.
```

Comment:
Templated `namespace` to be similar to what is found in the [uuidgen man](https://man7.org/linux/man-pages/man1/uuidgen.1.html), except also included the common namespaces inside the help message.